weboob/scripts/install.sh

```diff
@@ -2,4 +2,19 @@
 
 cd weboob
 rm -rf env
-mkdir -p ./env && virtualenv ./env && source ./env/bin/activate && pip install -r requirements.txt && cd .. && ./weboob/scripts/test.sh
+
+# Let's find out if Python's virtualenv is installed and, if so, what name its binary has
+pyvenv=""
+if [[ -z $(type -p virtualenv) ]]; then
+	if [[ -z $(type -p virtualenv2) ]]; then
```

Comment:
I was unaware of the `type` command until today, and my shell doesn't have any `man` help for it. Can you please replace the two uses by `which`, that is more known and documented?